react/features/mobile/navigation/components/RootNavigationContainer.js

```diff
@@ -39,6 +40,7 @@ const RootNavigationContainer = ({ isWelcomePageAvailable }: Props) => {
 
     return (
         <SafeAreaProvider>
+            <SafeAreaInsetsDetector />
```

Comment:
I think this should go together with DimensionsDetector no? In fact, I wonder if we could combine them to avoid having 2 components which measure screen space.

Reply:
Yep, the last bit is what I was saying.